mkosi/distributions/debian.py

```diff
@@ -245,10 +246,3 @@ def install_apt_sources(state: MkosiState, repos: Sequence[str]) -> None:
         with sources.open("w") as f:
             for repo in repos:
                 f.write(f"{repo}\n")
-
-    # Already contains a merged tree of repo_dirs after setup_apt
-    for src in state.workspace.joinpath("apt/sources.list.d").iterdir():
-        dst = state.root.joinpath("etc/apt/sources.list.d", src.name)
-        if dst.exists():
-            continue
-        shutil.copyfile(src, dst)
```

Comment:
Can we just drop this? This is writing to the root, which we don't do anywhere else?

Reply:
Thought about this already and wasn't entirely sure, but it shouldn't be a problem and the file can be easily passed in via some tree. Added it as an extra commit in case it needs reverting.